index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <span></span>
+    <link rel="stylesheet" href="static/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/feather-icons/dist/feather.min.js"></script>
+    <script
+      src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+      integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+      integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <div class="header">
+      <h1>Bot Controller</h1>
+      <div class="botSelector" style="width: 200px;">
+        <select id="botSelected" class="custom-select">
+          <option selected>Select Bot</option>
+          <option value="1">Boomer</option>
+          <option value="2">nickbot</option>
+        </select>
+      </div>
+      <div class="toggle-dark">
+        <div class="toggle">
+          <input
+            type="button"
+            class="btn btn-primary"
+            value="toggle theme"
+            onclick="toggleTheme();"
+          />
+          <label for="toggle"></label>
+        </div>
+      </div>
```

Comment:
Instead of a toggle for dark mode, we could just use `prefers-color-scheme`. Especially since it gets reset on every page reload, it would be much better to have it resort to the user's browser.

Reply:
thats a pretty good idea, but I think there should still be a toggle for dark mode

Reply:
i like this but still a toggle. i’m thinking we do the toggle storage with cookies maybe

Reply:
I think that we should have a switch that is set to the browser's preference on load, but can still be toggle.

Reply:
It's not really a good idea to have too many switches, so what might be the best idea is to just store the latest theme in cookies, and to keep the switch to toggle themes.